components/public-api/typescript/src/gitpod/experimental/v1/user_pb.ts

```diff
@@ -744,3 +744,4 @@ export class BlockUserResponse extends Message<BlockUserResponse> {
     return proto3.util.equals(BlockUserResponse, a, b);
   }
 }
+
```

Comment:
@mustard-mh why do we have such changes? I had something like that too, could you undo commits locally and try to commit again to make sure that git hooks are applied